app/scripts/components/exploration/components/layer-info-modal.tsx

```diff
@@ -134,14 +137,13 @@ export default function LayerInfoModal(props: LayerInfoModalProps) {
         />
       }
       footerContent={
-        <LinkComponent to={dataCatalogPath}>
+        <LinkComponent {...path}>
           <USWDSButton
             onClick={close}
             type='button'
             size='small'
             inverse={true}
             outline={false}
-            tabindex='-1'
```

Comment:
Any specific reason you got rid of this line? It looks like this component captures tab event two times without `tabindex='-1'`.